UI/js-src/lsmb/users/ChangePassword.js

```diff
@@ -113,12 +113,6 @@ define([
          submit_form: function () {
             var I = this;
             var r = request;
-            console.log("change_pw clicked");
-            var old_password = I.oldpw.get("value");
```

Comment:
@ylavoie I understand the removal of the first line, but the rest? Isn't that an error?

Reply:
Yes there is. Fixing

Reply:
It is fixed in a later commit in this PR so the end result is ok.